src/utils/auth.ts

```diff
@@ -50,10 +66,9 @@ export const login = async (
     window.localStorage.setItem('auth', verifyRes.data)
     window.localStorage.setItem('loggedInAddress', address)
     axiosInstance.defaults.headers.common['auth'] = verifyRes.data
-    window.location.reload()
     return verifyRes
   } catch (error) {
-    console.error('ERROR', error)
+    throw error
```

Comment:
Not a big deal, but if you just want to throw the caught error, you can just get rid of the try-catch block.

Reply:
Fixed